.mergify.yml

```diff
@@ -46,20 +50,14 @@ pull_request_rules:
           - "{{ author }}"
         branches:
           - "7.12"
-  - name: automatic merge for 7.x when CI passes
+        labels:
+          - "backport"
+  - name: automatic merge for 7\. branches when CI passes
     conditions:
       - check-success=fleet-server/pr-merge
       - check-success=CLA
-      - base=7.x
-    actions:
-      merge:
-        method: squash
-        strict: smart+fasttrack
-  - name: automatic merge for 7.12 when CI passes
-    conditions:
-      - check-success=fleet-server/pr-merge
-      - check-success=CLA
-      - base=7.12
+      - base~=^7\.
+      - label=backport
```

Comment:
I tested in one of my repositories an this condition filters as expected

![image](https://user-images.githubusercontent.com/2871786/115678682-ad474000-a349-11eb-874a-2eea8b0dda12.png)

https://github.com/v1v/apm-pipeline-library/pull/27/checks?check_run_id=2408046983

Reply:
With the author's condition:

![image](https://user-images.githubusercontent.com/2871786/115679599-a836c080-a34a-11eb-9276-ebaf991bd04c.png)